web_app/database/crud.py

```diff
@@ -0,0 +1,58 @@
+from typing import Any
+
+from sqlalchemy.orm import Session
+
+from utils.values import NotificationValidationValues
+
+
+def _exists_in_db(
+    db: Session = None,
+    model: Any = None,
+    attr: Any = None,
+    obj: Any = None,
+) -> bool:
+    """
+    Checks if the given attribute value already exists in the database
+    :param db: Session = Depends(get_database)
+    :param model: Any = None
+    :param attr: attr: Any = None
+    :param obj: Any = None
+    :return: bool
+    """
+    return (
+        db.query(model).filter(getattr(model, attr) == getattr(obj, attr)).first()
+        is not None
+    )
+
+
+def validate_fields(db: Session = None, obj: Any = None, model: Any = None) -> dict:
+    """
+    Validates all fields in the object a dict with exceptions if they were occured
+    :param db: Session = Depends(get_database)
+    :param obj: Any = None
+    :param model: Any = None
+    :return: dict
+    """
+    exceptions_dict = dict()
```

Comment:
`exception_dict` is not good naming, you are not raising any exception. 
It would be better to have `error_validation_dict`